packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -104,32 +117,37 @@ library ChainAssertions {
     }
 
     /// @notice Asserts that the L1CrossDomainMessenger is setup correctly
-    function checkL1CrossDomainMessenger(Types.ContractSet memory _contracts, Vm _vm, bool _isProxy) internal view {
-        IL1CrossDomainMessenger messenger = IL1CrossDomainMessenger(_contracts.L1CrossDomainMessenger);
+    function checkL1CrossDomainMessenger(IL1CrossDomainMessenger _messenger, Vm _vm, bool _isProxy) internal view {
         console.log(
             "Running chain assertions on the L1CrossDomainMessenger %s at %s",
             _isProxy ? "proxy" : "implementation",
-            address(messenger)
+            address(_messenger)
         );
-        require(address(messenger) != address(0), "CHECK-L1XDM-10");
+        require(address(_messenger) != address(0), "CHECK-L1XDM-10");
 
         // Check that the contract is initialized
-        DeployUtils.assertInitialized({ _contractAddress: address(messenger), _isProxy: _isProxy, _slot: 0, _offset: 20 });
+        DeployUtils.assertInitialized({
+            _contractAddress: address(_messenger),
+            _isProxy: _isProxy,
+            _slot: 0,
+            _offset: 20
+        });
 
         if (_isProxy) {
-            require(address(messenger.OTHER_MESSENGER()) == Predeploys.L2_CROSS_DOMAIN_MESSENGER, "CHECK-L1XDM-20");
```

Comment:
I see these checks removed from here and from `DeployImplementations.s.sol`, but I don't see them added anywhere. Is the plan to do that in a separate PR? It feels safer to move the checks in the same PR